themes/classic/templates/contact.tpl

```diff
@@ -26,10 +26,8 @@
 
 {block name='page_header_container'}{/block}
 
-{block name='left_column'}
-  <div id="left-column" class="col-xs-12 col-sm-4 col-md-3">
+{block name='column_contact'}
```

Comment:
I don't know the layout architecture very well, but this seems like a BC, if someone has overridden this layout in a them then the block name is not the same any more

Reply:
If we take in consideration it's a bc if we change template. We can't  fix css / js / tpl files :thinking: 
Since these files are not related to the `public API`, it's not a BC. ping @eternoendless

Reply:
It's not related to the API but it can break any child theme of classic no? Maybe we don't have a choice, it already happened I'm just wondering if there is an alternative solution and if not we will have to warn about this BC

Reply:
Is it possible to detect which layout is used? And adapt this template rather than modifying the generic ones?

Reply:
Maybe using the `$layout` variable that is used in `page.tpl`?

Reply:
Everytime we add an html element or a css property we potentially break the child theme :trollface: 
I don't know if there is a way to detect a layout :/